prime-router/src/test/resources/fhirengine/smoketest/Expected_FHIR_to_FHIR_FULLELR.fhir

```diff
@@ -1 +1,15159 @@
ctureDefinition/namespace-id","valueString":"M12776123.3"},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id","valueString":"2.16.840.1.114222.4.1.144"},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type","valueCode":"ISO"}]}],"type":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/v2-0203","code":"PLAC"}]},"value":"09339017"},{"extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority","extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id","valueString":"M12776123.3"},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type","valueCode":"ISO"}]}],"type":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/v2-0203","code":"FILL"}]},"value":"21:AA:B0029251S.3"},{"extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority","extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id","valueString":"M12776123"},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id","valueString":"2.16.840.1.114222.4.1.144"},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type","valueCode":"ISO"}]}],"type":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/v2-0203","code":"PGN"}]},"value":"21:AA:B0029251S"},{"extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority","extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id","valueString":"M12776123"},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id","valueString":"2.16.840.1.114222.4.1.144"},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type","valueCode":"ISO"}]}],"type":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/v2-0203","code":"FGN"}]},"value":"21:AA:B0029251S"}],"basedOn":[{"reference":"ServiceRequest/1713310292295952000.6b469277-2cc9-4ef2-83f2-021214d4f1a6"}],"status":"preliminary","subject":{"reference":"Patient/1713310292174793000.d3e32048-5cb6-46e5-9069-4ca92e4ac1f6"},"encounter":{"reference":"Encounter/1713310292182502000.7bd38229-d4b8-4ad9-9890-1aec968cc276"},"effectivePeriod":{"start":"2028-08-02T02:52:01-06:00","_start":{"extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time","valueString":"20280802025201-0600"}]},"end":"2028-08-02T02:52:01-06:00","_end":{"extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time","valueString":"20280802025201-0600"}]}},"issued":"2021-08-10T06:25:00-06:00","_issued":{"extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time","valueString":"20210810062500-0600"}]},"specimen":[{"reference":"Specimen/1713310292263224000.316e7c7b-d72b-42dd-ac9a-ba1455d1f0ca"},{"reference":"Specimen/1713310292256704000.770742e3-5895-4236-a0cc-adb40d08049b"}],"result":[{"reference"
```

Comment:
This change is mostly around updating the formatting to be more readable. There were a couple missing extension fields which were enough for the data comparison tool to log it out, but not enough for it to fail the tests. These extensions were unrelated to this change.